src/lib/charts/bar-chart.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+  import type { ChartData, ChartOptions, ChartItem } from 'chart.js';
+  import Chart from './chart.svelte';
+  export let data: ChartData<'bar'>, options: ChartOptions<'bar'>, chartID: ChartItem;
```

Comment:
```suggestion
  export let data: ChartData<'bar'>;
  export let options: ChartOptions<'bar'>;
  export let chartID: ChartItem;
  ```

Reply:
For readability, let's avoid multiple declarations on the same line.

Reply:
https://github.com/svelte-hackathon/data-visualization/pull/2/commits/32815f0542844eab77231ff7b6e9bea7a8658692